src/error-table.rkt

```diff
@@ -75,6 +162,13 @@
                                     subexpr-val)))
          (define cond-y (bfabs (bf/ y
                                     subexpr-val)))
+
+         (define x.eps (+ 127 (bigfloat-exponent x)))
+         (define y.eps (+ 127 (bigfloat-exponent y)))
```

Comment:
Where does 127 come from? I think it should be `(bf-precision x)` and similar for `y`.

Reply:
I think 127 is the exponent bias. I remember trying both this and bf-precision but only this got me the exact exponent (this was equivalent to floor(log2(x)))